### PR TITLE
added data to 24 languages for case insensitive identifier feature

### PR DESCRIPTION
**Pull request by mahadwaseem123**
Hi @breck7 and @celtic-coder , this refers to #313 . Kindly have a look. Thanks.

### CODE REVIEW

File: database/things/kotlin.pldb [hunk 1]
```diff
@@ -39,6 +39,7 @@ features
  hasMacros false
  hasOperatorOverloading true
  hasSemanticIndentation false
+ hasCaseInsensitiveIdentifiers false
```
Comment (celtic-coder) [inline database/things/kotlin.pldb:42]: Is this correct? I could not find anything on https://en.wikipedia.org/wiki/Kotlin_(programming_language), but it appears that it is case-insensitive.

File: database/things/cobol.pldb [hunk 2]
```diff
@@ -19,6 +19,7 @@ features
   MOVE ABC to XYZ  *> This is a comment.
  hasPrintDebugging true
  hasSemanticIndentation false
+ hasCaseInsensitiveIdentifiers true
```
Comment (celtic-coder) [inline database/things/cobol.pldb:22]: Not according to https://rosettacode.org/wiki/Case-sensitivity_of_identifiers#COBOL

File: database/things/kotlin.pldb (hunk 1 above)
Comment (celtic-coder) [reply database/things/kotlin.pldb:42]: https://rosettacode.org/wiki/Case-sensitivity_of_identifiers#Kotlin says that it is case-sensitive.

File: database/things/php.pldb [hunk 3]
```diff
@@ -100,6 +100,7 @@ features
   /* A comment
   */
  hasSemanticIndentation false
+ hasCaseInsensitiveIdentifiers true
```
Comment (celtic-coder) [inline database/things/php.pldb:103]: https://en.wikipedia.org/wiki/Case_sensitivity#In_programming_languages says: 
> in [PHP](https://en.wikipedia.org/wiki/PHP), for example, variable names are case-sensitive but function names are not case-sensitive

File: database/things/postgresql.pldb [hunk 4]
```diff
@@ -24,6 +24,7 @@ features
   /* A comment
   */
  hasSemanticIndentation false
+ hasCaseInsensitiveIdentifiers false
```
Comment (celtic-coder) [inline database/things/postgresql.pldb:27]: Again, https://en.wikipedia.org/wiki/Case_sensitivity#In_programming_languages says:
> Others are case-insensitive (i.e., not case-sensitive), such as... [SQL](https://en.wikipedia.org/wiki/SQL) (for the syntax)

According to https://en.wikipedia.org/wiki/PostgreSQL#Standards_compliance, it seems to be case-insensitive.

File: database/things/cobol.pldb (hunk 2 above)
Comment (mahadwaseem123) [reply database/things/cobol.pldb:22]: Hi @celtic-coder , it is not case sensitive according to https://devops.com/the-beauty-of-the-cobol-programming-language-v2/#:~:text=FAST%20FACT%3A%20Is%20COBOL%20a,named%20ws%2Dfirst%2Dname.

File: database/things/php.pldb (hunk 3 above)
Comment (mahadwaseem123) [reply database/things/php.pldb:103]: So @celtic-coder , do we mark it false in this case?

File: database/things/postgresql.pldb (hunk 4 above)
Comment (mahadwaseem123) [reply database/things/postgresql.pldb:27]: @celtic-coder , same scenario as php but most of the websites suggest its case sensitive. What you suggest? I will change accordingly.

File: database/things/kotlin.pldb (hunk 1 above)
Comment (mahadwaseem123) [reply database/things/kotlin.pldb:42]: Suggest about it too @celtic-coder

**Comment by mahadwaseem123** [comment]
Thanks for the detailed review @celtic-coder

File: database/things/cobol.pldb (hunk 2 above)
Comment (celtic-coder) [reply database/things/cobol.pldb:22]: From what I've seen, the Rosetta Code example may not be completely correct. Other examples, such as https://www.geeksforgeeks.org/variables-in-cobol/ seem to indicate that case does not matter. Let's go with case-insensitive.

File: database/things/php.pldb (hunk 3 above)
Comment (celtic-coder) [reply database/things/php.pldb:103]: Yes, "false" for PHP.

File: database/things/postgresql.pldb (hunk 4 above)
Comment (celtic-coder) [reply database/things/postgresql.pldb:27]: I've written lots of SQL code in both the Microsoft and Oracle varieties and on these platforms it is case-insensitive. I would imagine that PostgreSQL would be the same. The "Standards Compliance" section of the Wikipedia article would seem to indicate this also.

**Comment by mahadwaseem123** [comment]
Hi @celtic-coder @breck7 , made the changes. Kindly review and merge. Thanks.

**Comment by celtic-coder** [comment]
Great work, Mahad (@mahadwaseem123) -- thanks for following up! 👍

**Pull request merged**